odysseus/convertFtbQuests.ts

```diff
@@ -478,6 +513,8 @@ function convertTask(task: QuestTask, questFile: QuestFile): HeraclesQuestTask {
         case "checkmark": {
```

Comment:
Since the `title` and `icon` here are shared between many types, why not make them into a variable then spread them into the individual types? like `...sharedData` or something similar

Reply:
done in 644b643